### ISSUE TITLE
InterceptAcknowledgedMessage.StoredMessage is inaccessible

### ISSUE THREAD

**Issue by austinw-fineart** [moquette-io/moquette#444]
InterceptAcknowledgedMessage.StoredMessage is inaccessible because its visibility is package-private. Therefore on InterceptHandler's, InterceptAcknowledgedMessage's StoredMessage cannot be accessed including its methods such as getPayload().

### Moquette MQTT version
0.12

### JVM version (e.g. `java -version`)
11.0.1

### OS version (e.g. `uname -a`)
Windows 10 1809

**Comment by noordawod** [comment]
@andsel Any chance to fix this?

**Comment by noordawod** [comment]
@andsel any chance to fix?

**Comment by andsel** [comment]
I'll fix it, but I still thinking which is the best way. StoredMessage is an internal class used to serialize messages on H2 values, so the decision is between augment visibility of StoredMessage to public or provide another class that's not used internally by the broker.
It will be fixed with version 0.13

**Comment by noordawod** [comment]
Right. If it's indeed an internal property then it sounds like a good idea to keep it private. But this model may be of use outside Moquette, so having a public one is also good. Thanks for your time and efforts :)
